data/gala@wayland.service.in

```diff
@@ -0,0 +1,36 @@
+[Unit]
+Description=Gala on Wayland
+# On wayland, force a session shutdown
+OnFailure=gnome-session-shutdown.target
+OnFailureJobMode=replace-irreversibly
+CollectMode=inactive-or-failed
+RefuseManualStart=on
+RefuseManualStop=on
+
+After=gnome-session-manager.target
+
+Requisite=gnome-session-initialized.target
+PartOf=gnome-session-initialized.target
+Before=gnome-session-initialized.target
+
+#NOTE: ConditionEnvironment works with systemd >= 246
+ConditionEnvironment=XDG_SESSION_TYPE=%I
+
+[Service]
+Slice=session.slice
+Type=notify
+ExecStart=@bindir@/gala
+# Exit code 1 means we are probably *not* dealing with an extension failure
+SuccessExitStatus=1
```

Comment:
I think this is unecessary, We don't have extensions.

Reply:
Indeed